solidity/ecdsa/contracts/libraries/EcdsaAuthorization.sol

```diff
@@ -31,18 +31,30 @@ library EcdsaAuthorization {
         // authorization decrease amount is small, significant, or if it is
         // a decrease to zero.
         uint64 authorizationDecreaseDelay;
+        // Reserved storage space in case we need to add more variables.
+        // See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+        // slither-disable-next-line unused-state
+        uint256[18] __gap;
```

Comment:
@cygnusv I'm curious about your opinion, whether it makes sense to add gaps to structs that are used as a storage for libraries?

Reply:
Posted a question on OpenZeppelin's forum - [link](https://forum.openzeppelin.com/t/storage-gaps-in-libraries-and-structs/28171).